components/src/Button/Button.js

```diff
@@ -122,6 +136,9 @@ const Button = styled.button`
     pointer-events: none;
   }
 
+  * {
```

Comment:
I'm going to update this PR to wrap the children in a span to make this stying a little nicer